daemon-tests/src/flow.rs

```diff
@@ -39,15 +40,9 @@ pub async fn next_maker_offers(
     Ok((a?, b?))
 }
 
-pub async fn is_next_offers_none(
-    rx: &mut OffersFeed,
-    contract_symbol: &ContractSymbol,
-) -> Result<bool> {
-    let mut rx = match contract_symbol {
-        ContractSymbol::BtcUsd => rx.btc_usd.clone(),
-        ContractSymbol::EthUsd => rx.eth_usd.clone(),
-    };
-    let maker_offers = next(&mut rx).await?;
+// TODO: what is this function even doing? only called from the tests and the response is ignored.
```

Comment:
❓ @luckysori @da-kami do you know what this function is used for? It is only used by the tests, but the result is alwas ignored. I would like to remove it, but am not sure if this code is hidding some kind of magic :)